### ISSUE TITLE
polylog2(2.25 + 0.0im) =/= polylog2(2.25)

### ISSUE THREAD

**Issue by barton-willis** [barton-willis/polylog#5]
We have

~~~
polylog2(2.25 + 0.0im)
2.454029219793818 + 2.547612409839201im
~~~

But

~~~
polylog2(2.25)
2.454029219793818 - 2.547612409839201im
~~~

For `x > 1`, the imaginary part of `Li_2(x + im y)` has a jump discontinuity at zero.  I'm not sure if ` y  -->  Li_2(x + im y)` is continuous from above or below.  But I have some evidence that `polylog2(2.25) = 2.454029219793818 - 2.547612409839201im` is correct.

**Comment by barton-willis** [comment]
OK, I think what happens is this

~~~
clog(-2.25)
0.8109302162163288 + 3.141592653589793im

clog(-2.25 + 0.0im)
0.8109302162163288 - 3.141592653589793im

clog(-2.25 - 0.0im)
0.8109302162163288 - 3.141592653589793im
~~~

**Comment by barton-willis** [comment]
Fixed by 2397b34ce6d8746d68132cea8050ac9acc8742ce
